corehq/apps/cloudcare/static/cloudcare/js/form_entry/fullform-ui.js

```diff
@@ -641,6 +641,8 @@ Formplayer.Errors = {
         "Internet in order to submit your form.",
     LOCK_TIMEOUT_ERROR: gettext('Another process prevented us from servicing your request. ' +
         'Please try again later.'),
+    NO_INTERNET_ERROR: gettext("We have detected an issue with your network. " +
+        "Please check your Internet connection"),
```

Comment:
Do you think it's worth explicitly telling them they can retry once they have connectivity?

Reply:
yeah. that might be worth it.